### PR TITLE
thepiratebay: add minimumratio and minimumseedtime

### PR DESCRIPTION
**Pull request by ngosang**

### CODE REVIEW

File: src/Jackett.Common/Indexers/ThePirateBay.cs [hunk 1]
```diff
@@ -232,6 +229,8 @@ private ReleaseInfo CreateReleaseInfo(QueryResponseItem item)
                 Author = item.Username,
                 DownloadVolumeFactor = 0,
                 UploadVolumeFactor = 1,
+                MinimumRatio = 1,
+                MinimumSeedTime = 172800, // 48 hours
```
Comment (garfield69) [inline src/Jackett.Common/Indexers/ThePirateBay.cs:233]: Huh?
its a public site so there is no requirement to seed and no hit and run penalties.
Generally only private sites (and some few semi private ones) that have these.
I do not believe public sites should have MR and MST in their indexers.

File: src/Jackett.Common/Indexers/ThePirateBay.cs [hunk 2]
```diff
@@ -17,9 +17,6 @@
 
 namespace Jackett.Common.Indexers
 {
-    /// <summary>
-    /// The Pirate Bay via API.
-    /// </summary>
```
Comment (garfield69) [inline src/Jackett.Common/Indexers/ThePirateBay.cs:22]: why is the summary removed?

File: src/Jackett.Common/Indexers/ThePirateBay.cs (hunk 1 above)
Comment (ngosang) [reply src/Jackett.Common/Indexers/ThePirateBay.cs:233]: All public trackers have those values in Jackett. We can remove them all public trackers if you want to.

File: src/Jackett.Common/Indexers/ThePirateBay.cs (hunk 2 above)
Comment (ngosang) [reply src/Jackett.Common/Indexers/ThePirateBay.cs:22]: This is not meaningful. The class is called ThePirateBay and from my experience if the tracker changes the name or the implementation the comment will be outdated and will be misleading.

File: src/Jackett.Common/Indexers/ThePirateBay.cs (hunk 1 above)
Comment (garfield69) [reply src/Jackett.Common/Indexers/ThePirateBay.cs:233]: Yes, public trackers should either not have MR and MST or perhaps they should be set to zero. (I favour not having any).
None of the Cardigann public indexers have MR and MST unless they were once private and became public and someone forgot to remove them.

**Comment by garfield69** [comment]
I just checked the Cardigann indexers and none of the public have MR and/or MST 
I'll review the public C# ones next and update as required.

**Comment by ngosang** [comment]
@garfield69 There are default values for Cardigann indexers (if you don't set them in the yml) => https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Indexers/CardigannIndexer.cs#L1424
I can remove those defaults but I don't know if there is a reason to be there. Are those fields are mandatory in Torznab specification or something?

**Comment by garfield69** [comment]
>There are default values for Cardigann indexers (if you don't set them in the yml) => https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Indexers/CardigannIndexer.cs#L1424
I can remove those defaults but I don't know if there is a reason to be there. Are those fields mandatory in Torznab specification or something?

Thanks, but I've already removed those defaults just now, which were added in error 9 months ago after a miss-communication ;-).
MR and MST are not mandatory Torznab requirements, and should only be present when a web site enforces a minimum ratio and seed time to avoid a users' account being flagged as a Hit and Run violators which can lead to a loss of privileges or even account suspension/deletion.
This usually applies only to private sites, when they specify these requirements in their Rules section.
Public sites don't have any such requirements.

**Pull request closed without merging**